docs/release-notes/checkpoint-prefix.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Kubernetes: Specifying `checkpointStorage.prefix` for `checkpointStorage.type=s3` will now upload
+   checkpoints correctly while before it was ignored.
```

Comment:
nit: I would call this a feature instead of a bug fix.

Also how about: "Kubernetes: Users may now specify a `checkpointStorage.prefix` in the Determined helm chart if using s3 buckets for checkpoint storage. Checkpoints will now be uploaded with the path prefix whereas before it was ignored."